test/e2e_appium/pages/onboarding/seed_phrase_input_page.py

```diff
@@ -0,0 +1,276 @@
+"""
+Seed Phrase Input Page for Status Desktop E2E Testing
+
+Page object for the seed phrase input screen during onboarding.
+Supports importing existing seed phrases for account recovery.
+"""
+
+import logging
+import time
+from typing import List, Union
+from ..base_page import BasePage
+from locators.onboarding.seed_phrase_input_locators import SeedPhraseInputLocators
+
+
+class SeedPhraseInputPage(BasePage):
+    """Page object for the Seed Phrase Input Screen during onboarding"""
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.locators = SeedPhraseInputLocators()
+        self.logger = logging.getLogger(__name__)
```

Comment:
```suggestion
        self.logger = logging.getLogger(__name__)
```

move this to the base class , its constantly repetitive line